### PR TITLE
Simplify

### PR DESCRIPTION
**Pull request by Muennighoff**

### CODE REVIEW

File: mteb/abstasks/MultiSubsetLoader.py [hunk 1]
```diff
@@ -9,8 +9,7 @@ def load_data(self, **kwargs):
         if self.data_loaded:
             return
 
-        fast_loading = self.fast_loading if hasattr(self, "fast_loading") else False
-        if fast_loading:
+        if hasattr(self, "fast_loading"):
```
Comment (KennethEnevoldsen) [inline mteb/abstasks/MultiSubsetLoader.py:12]: if could be false as well I assume?

**Review by KennethEnevoldsen — CHANGES_REQUESTED** [review]
looks good a pull fr

File: mteb/abstasks/MultiSubsetLoader.py (hunk 1 above)
Comment (Muennighoff) [inline mteb/abstasks/MultiSubsetLoader.py:12]:
```suggestion
        if hasattr(self, "fast_loading") and self.fast_loading:
```

Comment (Muennighoff) [reply mteb/abstasks/MultiSubsetLoader.py]: Sorry fixed it!

**Pull request merged**